_tour/generic-classes.md

```diff
@@ -31,7 +31,7 @@ class Stack[A] {
 ```
 This implementation of a `Stack` class takes any type `A` as a parameter. This means the underlying list, `var elements: List[A] = Nil`, can only store elements of type `A`. The procedure `def push` only accepts objects of type `A` (note: `elements = x :: elements` reassigns `elements` to a new list created by prepending `x` to the current `elements`).
 
-`Nil` here is a special case of `List` and is not to be confused with `Null`. It is a `List` of zero length.
+`Nil` here is a special case of `List` and is not to be confused with `Null`. It is simply an empty list.
```

Comment:
Maybe just "`Nil` here is the empty `List` (not to be confused with `null`)." is enough here.

Reply:
Same difference, IMO, but now changed